storage/innobase/handler/ha_innodb.cc

```diff
@@ -10948,15 +10948,19 @@ create_table_info_t::create_table_def()
 
 	heap = mem_heap_create(1000);
 
+    bool have_vers_start = false, have_vers_end = false;
```

Comment:
We do not want `-Wunused` warnings for the non-debug build. Please write conditional code as follows:
```c++
ut_d(bool have_vers_start = false);
ut_d(bool have_vers_end = false);
```
And be sure to indent the code in the same way as the preceding and succeeding lines.

Reply:
Sorry, had a fresh IDE installation last month, not yet configured for innodb